pkg/controller/storagecluster/uninstall_reconciler.go

```diff
@@ -593,3 +600,28 @@ func (r *ReconcileStorageCluster) deleteSnapshotClasses(instance *ocsv1.StorageC
 	}
 	return nil
 }
+
+func (r *ReconcileStorageCluster) deleteQuickStarts(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+	for _, qs := range AllQuickStarts {
+		cqs := consolev1.ConsoleQuickStart{}
+		err := yaml.Unmarshal(qs, &cqs)
+		if err != nil {
+			reqLogger.Info(fmt.Sprintf("failed to unmarshal consolequickstarts, name: %s, namespace: %s", cqs.Name, cqs.Namespace))
```

Comment:
1. cqs will be invalid if the call to Unmarshal() fails. Use qs to identify the failure.
2. Instead of using fmt.Sprintf() , use the recommended method of printing fields:
reqLogger.Info("Failed to unmarshal consolequickstarts", "FieldName", value)

Reply:
1. will sprint the string which failed the unmarshal if thats ok 
2. i find fmt.sprintf to be much more readable but i've changed it.

Reply:
> 1. will sprint the string which failed the unmarshal if thats ok
That is fine - my concern was that cqs would be invalid in this case.